INSTALL.md

```diff
@@ -163,6 +184,12 @@ Next, make a test environment:
 qiita-env make --no-load-ontologies
 ```
 
+Finally, redbiom rely on the REDBIOM_HOST environment variable to set the URL to query. By default is set to http://127.0.0.1:7379, which is the webdis default. For example you could:
```

Comment:
rely -> relies

once we shift to a release, it's be super awesome to have a default `REDBIOM_HOST` for clients, and that its `qiita.ucsd.edu` with forwarding provided by nginx.